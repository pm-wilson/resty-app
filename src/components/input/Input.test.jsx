// import React from 'react';
// import { render, cleanup } from '@testing-library/react';
// import Input from './Input';

// describe('Input component', () => {
//   afterEach(() => cleanup());
//   it('renders Input', () => {
//     const { asFragment } = render(<Input />);
//     expect(asFragment()).toMatchSnapshot();
//   });
// });
