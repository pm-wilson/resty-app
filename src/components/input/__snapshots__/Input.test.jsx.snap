// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component renders Input 1`] = `
<DocumentFragment>
  <div>
    <form>
      <div
        class="methodradio"
      >
        <label>
          <span>
            GET
          </span>
          <input
            id="get"
            name="method"
            type="radio"
            value="GET"
          />
        </label>
        <label>
          <span>
            POST
          </span>
          <input
            id="post"
            name="method"
            type="radio"
            value="POST"
          />
        </label>
        <label>
          <span>
            PUT
          </span>
          <input
            id="put"
            name="method"
            type="radio"
            value="PUT"
          />
        </label>
        <label>
          <span>
            PATCH
          </span>
          <input
            id="patch"
            name="method"
            type="radio"
            value="PATCH"
          />
        </label>
        <label>
          <span>
            DELETE
          </span>
          <input
            id="delete"
            name="method"
            type="radio"
            value="DELETE"
          />
        </label>
      </div>
      <label
        class="addresssearch"
      >
        Address Search
        <input
          id="url"
          name="url"
          type="text"
          value="www.google.com"
        />
      </label>
      <label
        class="jsonbody"
      >
        JSON Body
        <input
          id="body"
          name="body"
          type="text"
          value=""
        />
      </label>
      <button
        class="submitbutton"
      >
        Submit
      </button>
    </form>
  </div>
</DocumentFragment>
`;
